core/conversion/InterfaceTypes.cpp

```diff
@@ -11,7 +11,7 @@ GraphParams get_named_params(c10::ArrayRef<torch::jit::Value*> inputs, std::vect
   GraphParams named_params;
   auto param_it = params.begin();
   for (auto in : inputs) {
-    if (in->type() != c10::TensorType::get() && in->isCompleteTensor() && param_it != params.end()) {
+    if (in->type() != c10::TensorType::get() && param_it != params.end()) {
```

Comment:
Did this end up being fixed?

Reply:
Facing this error
```
Unexpected specifier 'strides':
```
Looks like the strides specifier did not make it into 1.7 branch. [General branch stride support](https://github.com/pytorch/pytorch/blob/5741de883af2ac63cec537fc8a554610a5cea853/torch/csrc/jit/frontend/schema_type_parser.cpp#L225), [1.7 branch code comparison](https://github.com/pytorch/pytorch/blob/1.7/torch/csrc/jit/frontend/schema_type_parser.cpp#L244)
From my understanding, the isCompleteTensor() doesn't seem to be absolutely necessary especially when we pass in constants to the graph assuming they have static shapes. The testcase usages don't absolutely need it. 
Can you let me know if you have any other necessary usecases in mind where this would help ? 
This definitely adds a extra layer of check which is good. I tried looking into pytorch 1.7 source and building it to understand why `isCompleteTensor` function changed, however that was blocked by some unrelated issues.